data/yam/agama/auto/default_sle.sh

```diff
@@ -0,0 +1,7 @@
+set -ex
```

Comment:
this is from one year ago, we are not still testing with .sh files, you don't need it.

Reply:
a36152b8a